### ISSUE TITLE
Sigmoid Function on the output Layer ?

### ISSUE THREAD

**Issue by amithadiraju1694** [jinudaniel/neural_recommender#1]
Hey @jinudaniel,

First of all, thank you so much for such an easily interpretative implementation of the paper. I'm a bit confused on one part though.

 Your implementations: _**Model1**_ and _**Model2**_ made complete sense to me, but _**Model3**_ is where i'm a bit fuzzy. To me this is more or less a regression problem, i.e., trying to predict movie ratings based on previous interactions, the loss function of _'mse'_ along with _'relu'_ or even linear activation's make sense, but how come a _**'sigmoid'**_ function is used for activation on the last layer ?

Wouldn't the sigmoid function always lead to outputs between [0,1] ? Even if we perform 'N' number of hyper-parameter tuning steps  and other regularization techniques , technically a sigmoid function never crosses an output of 1.0 right ? I looked at other implementations of this paper and pretty much found the same thing, a sigmoid function at the end even for a regression problem. 

I'm not trying to contradict your or the original authors idea here, I'm just trying to figure out how sigmoid activation would make sense for this problem. Or is there any piece I'm missing fundamentally here. Let me what you think.
